types/gulp-strip-comments/gulp-strip-comments-tests.ts

```diff
@@ -0,0 +1,12 @@
+import * as gulp from 'gulp';
+import * as stripComments from 'gulp-strip-comments';
```

Comment:
An ES6 module namespace object cannot be invoked as a function. Use `import stripComments = require("gulp-strip-comments")` instead.